### PR TITLE
Signal policy publish operation completion to the service

### PR DESCRIPTION
**Pull request by hausdorff**
Fixes pulumi/pulumi-policy#32

### CODE REVIEW

File: pkg/backend/httpstate/client/client.go [hunk 1]
```diff
@@ -472,6 +479,16 @@ func (pc *Client) PublishPolicyPack(ctx context.Context, orgName string,
 		return errors.Wrapf(err, "Failed to upload compressed PolicyPack")
 	}
 
+	//
+	// Step 3: Signal to the service that the PolicyPack publish operation is complete.
+	//
+
+	err = pc.restCall(ctx, "POST",
```
Comment (ekrengel) [inline pkg/backend/httpstate/client/client.go]: nit: you could use `http.MethodPost`

Comment (ekrengel) [reply pkg/backend/httpstate/client/client.go]: Does this restCall take care of situations where we dont get a 200 back?

Comment (pgavlin) [reply pkg/backend/httpstate/client/client.go]: I think it will handle retry on 5xx responses. 4xx responses will be handled by returning an error.

Comment (hausdorff) [reply pkg/backend/httpstate/client/client.go]: I normally do this, but the rest of the client does not. So we do it this way to be consistent.

**Pull request merged**